src/cli/commands/help.js

```diff
@@ -14,45 +14,50 @@ export function hasWrapper(flags: Object, args: Array<string>): boolean {
 
 export function setFlags(commander: Object) {}
 
+function rawText(str: string): {inspect(): string} {
+  return {
+    inspect(): string {
+      return str;
+    },
+  };
+}
```

Comment:
Ok - what do you think about putting this in the BaseReporter class, so that it can become:

```js
reporter.log(reporter.lang('helpExamples', reporter.rawText(examples.join('\n'))));
```

It would become a tool whose implementation would be entirely contained into the reporter. Whether it's hacky or not would be something the commands would have to care or not.